openebs/openebs-provisioner.go

```diff
@@ -231,12 +226,14 @@ func main() {
 	flag.Parse()
 	flag.Set("logtostderr", "true")
 	var (
-		config *rest.Config
-		err    error
+		config     *rest.Config
+		err        error
+		k8sMaster  = mayav1.K8sMasterENV()
+		kubeConfig = mayav1.KubeConfigENV()
 	)
-	if *master != "" || *kubeconfig != "" {
-		config, err = clientcmd.BuildConfigFromFlags(*master, *kubeconfig)
-		fmt.Printf("Client config was built using flags: Address: '%s' Kubeconfig: '%s' \n", *master, *kubeconfig)
+	if len(k8sMaster) != 0 || len(kubeConfig) != 0 {
+		config, err = clientcmd.BuildConfigFromFlags(k8sMaster, kubeConfig)
+		fmt.Printf("Client config was built using flags: Address: '%s' Kubeconfig: '%s' \n", k8sMaster, kubeConfig)
```

Comment:
Remove the word flags.
Put the statement before invoking BuildConfigFromFlags, use appropriate tense!!
Check the error & handle the error

Reply:
@AmitKumarDas error is handled below the if-else statement.